src/main/java/io/cdap/e2e/pages/locators/CdfSignInLocator.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.e2e.pages.locators;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+/**
+ * Represents CdfSignInLocator
+ */
+public class CdfSignInLocator {
+
+    @FindBy(how = How.XPATH, using = "//input[@name='identifier']")
+    public WebElement cdfUsername;
+
+    @FindBy(how = How.XPATH, using = "//input[@name='Passwd']")
+    public WebElement cdfPassword;
+
+    @FindBy(how = How.XPATH, using = "//*[contains(text(),'Next')]//parent::button")
+    public WebElement nextButton;
+
+    public static By locatorOfEmailTextBox() {
+        return By.xpath("//input[@name='identifier']");
+    }
+
+    public By selectTestAccount() {
+        return By.xpath("//*[contains(text(),'CloudDataFution Automation')]");
```

Comment:
Create a dynamic locator. Read the Test Account name from the properties file.

Reply:
Updated [https://github.com/cloudsufi/cdap-e2e-tests/pull/2/commits/2fa8b80cc365818abeaf5bec5c89ab1185aa83dc#diff-b3632a67370578b810c74200a2a8673e2a0f17fefb10874d9bf696b660f6ca0aR17]